content/questions/actions/question-044.md

```diff
@@ -6,8 +6,8 @@ draft: false
 ---
 
 
-> https://docs.github.com/en/actions/using-workflows/storing-workflow-data-as-artifacts#about-workflow-artifacts
-1. [x] You cannot access `artifacts` that were created in a different workflow run
-1. [ ] Use the `actions/download-artifact` action.
+> https://docs.github.com/en/actions/using-workflows/storing-workflow-data-as-artifacts#downloading-or-deleting-artifacts
+1. [ ] You cannot access `artifacts` that were created in a different workflow run
+1. [x] Use the `actions/download-artifact` action.
```

Comment:
As stated in the screenshot you pointed out yourself - `actions/download-artifact` works only if the artifact was created in the same workflow run. The question asks about accessing artifacts from **previously triggered workflow run**.

As for me there is nothing to fix here. Please explain if I'm missing something

Reply:
Hi,
hm I think I misunderstood question or the question is so tricky to grab its meaning. I would put like: 
To access an `artifact` that was created in another workflow run you can:

But it is fine, thank you for clarifying